streamlit_aggrid_redux/grid_builder.py

```diff
@@ -102,6 +102,22 @@ def _process_css(custom_css: Union[Dict, str]) -> Dict:
         raise GridBuilderError(
             f"Custom CSS is neither a dict nor a string by {type(custom_css)}"
         )
+    
+def _process_update_on(updates: List[str], options: Dict) -> List[str]:
+    """ Ensure that updates include columnVisible and if we have checkboxes visible. """
```

Comment:
This function is what generates the fix for #13: it pushes the `selectionChanged` listener when `checkbox` is requested